test/spoom/cli/commands/bump_test.rb

```diff
@@ -0,0 +1,159 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'pathname'
+
+require_relative "../cli_test_helper"
+
+module Spoom
+  module Cli
+    module Commands
+      class BumpTest < Minitest::Test
+        include Spoom::Cli::TestHelper
+        extend Spoom::Cli::TestHelper
+
+        TEMPORARY_DIRECTORY = "temp"
```

Comment:
```suggestion
        TEMPORARY_DIRECTORY = "test-bump"
```

so we're sure it will never conflict with another test